Sources/InjectionNext/InjectionNext.swift

```diff
@@ -148,6 +148,19 @@ open class InjectionNext: SimpleSocket {
                 let dylib = NSTemporaryDirectory() + dylibName
                 try! data.write(to: URL(fileURLWithPath: dylib))
                 injectAndSweep(dylib)
+            case .metrics:
+                guard let metricsJSON = readString() else {
+                    return error("Unable to read metrics JSON")
+                }
+                if let data = metricsJSON.data(using: .utf8),
+                   let metricsDict = try? JSONSerialization.jsonObject(with: data) as? [String: Any] {
+                    let notificationName = (metricsDict["notification_name"] as? String) ?? "INJECTION_METRICS_NOTIFICATION"
```

Comment:
Why not extend the if so "INJECTION_METRICS_NOTIFICATION" isn't here? I guess it should also move into InjectionLite/Sources/InjectionImplC/include/InjectionImplC.h along with "INJECTION_BUNDLE_NOTIFICATION".